lib/fathead/caniuse/parse.py

```diff
@@ -31,15 +31,25 @@ def generate_answers(data):
         ])
         print ','.join(titles)
 
+        # Generate Related Topics
+        links = feature_data.get('links', [])
+        related = u' '.join([u'[[{} {}]]'.format(link['title'], link['url']) for link in links])
+
         # Generate abstract
         abstract = feature_data['description']
         for browser in ['ie', 'firefox', 'chrome', 'safari', 'ios_saf', 'android']:
             agent = data['agents'][browser]
             out = browser_support(
-                browser=agent['browser'], 
-                prefix=agent['prefix'], 
+                browser=agent['browser'],
+                prefix=agent['prefix'],
                 stats=feature_data['stats'][browser])
             abstract += '<br>' + out
+
+        # Add notes to abstract, if there are any
+        notes = feature_data.get('notes', '')
+        if notes:
+            abstract += '<br>' + notes
```

Comment:
@samskeller I'd like to make the notes a little more obvious and separate them from the other content.

Let's wrap the notes in their own `<p>` which will space things out a bit. We should also add "Notes:" above to make it clear. The final output should look like:

```html
<p><b>Notes:</b> $notes</p>
```

Also, it looks like the notes (and now also the descriptions) contain Markdown. We should either convert the markdown to HTML, or strip out the HTML.

Would you like to try converting the Markdown to HTML first? I believe that approach will work.